tests/library/test_loaders.py

```diff
@@ -68,6 +68,28 @@ def test_load_csv(self):
                 ):
                     self.assertEqual(saved_instance[1].to_dict(), loaded_instance)
 
+    def test_load_csv_with_pandas_args(self):
+        # Using a context for the temporary directory
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            files = {}
+            dfs = {}
+
+            for file in ["train", "test"]:
+                path = os.path.join(tmp_dir, file + ".csv")  # Adding a file extension
```

Comment:
not critical, but makes sense to call it tsv.

Reply:
done